pkg/virt-handler/notify-server/server.go

```diff
@@ -75,13 +75,15 @@ func (n *Notify) HandleDomainEvent(_ context.Context, request *notifyv1.DomainEv
 		}
 	}
 
-	log.Log.Object(domain).Infof("Received Domain Event of type %s", request.EventType)
 	switch request.EventType {
 	case string(watch.Added):
+		log.Log.Object(domain).Infof("Received Domain Event of type %s", request.EventType)
```

Comment:
I think you can give all log lines, except the one for error a verbosity bigger than 2.

Reply:
The reason I've only did that for `watch.Modified` is that in https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md it is suggested that for `V(2)` there should be logging for "significant changes in the system". And so I thought that it maybe makes sense to leave it for `watch.Added` and `watch.Deleted` but `watch.Modified` could be too much and I think it better plays out in `V(3)`.

Reply:
Ok, I think these log messages are too low-level to really indicate a "significant changes in the system". So it would be fine to use `V3`.